webcurator-ui/src/views/target/target-tabs/TargetTabAnnotations/TargetTabPanelAnnotations.vue

```diff
@@ -0,0 +1,178 @@
+const targetId = Number(route.params.id);
+
+const showNewAnnotationModal = () => {
+  newAnnotationModal.open( NewAnnotationModal, {
+    props: { header: 'New Annotation', modal: true, dismissableMask: true },
+    data: {
+      annotation: newAnnotation,
+      addAnnotation: addAnnotation
+    }
+  });
+}
+
+defineProps<{
+  editing: boolean;
+}>();
+
+const annotations = ref<Array<Annotation>>([]);
+const sortDirection = ref('desc');
+const targetAnnotations = useTargetAnnotationsDTO().targetAnnotations;
+const selectionTypes = ["Area", "Collection", "Other collections", "Producer type", "Publication type"];
+const harvestTypes = ["Event", "Subject", "Theme"];
+const userProfile = useUserProfileStore();
+const newAnnotation = ref(<Annotation>{ alert: false, user: userProfile.name });
+const loading = ref(false);
+
+async function prepareAnnotations() {
+  loading.value = true;
+  
+  const targetInstanceAnnotations = await useTargetInstanceListStore().getTargetInstanceAnnotations(targetId);
+
+  targetInstanceAnnotations.forEach((annotation: Annotation) => {
+    annotations.value.push(annotation);
+  });
+
+  targetAnnotations.annotations.forEach((annotation: Annotation) => {
+    annotations.value.push(annotation);
+  });
+
+  annotations.value.sort((a, b) => new Date(a.date).valueOf() - new Date(b.date).valueOf());
+
+  loading.value = false;
+}
+
+const sortAnnotaions = () => {
+  if (sortDirection.value === 'asc') {
+    annotations.value.sort((a, b) => new Date(a.date).valueOf() - new Date(b.date).valueOf());
+    sortDirection.value = 'desc';
+  } else {
+    annotations.value.sort((a, b) => new Date(b.date).valueOf() - new Date(a.date).valueOf());
+    sortDirection.value = 'asc';
+  }
+}
+
+const addAnnotation = () => {
+  newAnnotation.value.date = new Date().toISOString();
+  targetAnnotations.annotations.push(newAnnotation.value);
+  if (sortDirection.value === 'asc') {
+    annotations.value.unshift(newAnnotation.value);
+  } else {    
+    annotations.value.push(newAnnotation.value);
+  }
+  newAnnotation.value = <Annotation>{ alert: false, user: userProfile.name };
+}
+
+const deleteAnnotation = (annotation: Annotation) => {
+  targetAnnotations.annotations = targetAnnotations.annotations.filter((a) => 
+    !(a.date === annotation.date && a.user === annotation.user && a.note === annotation.note)
+  );
+  annotations.value = annotations.value.filter((a) =>
+    !(a.date === annotation.date && a.user === annotation.user && a.note === annotation.note)
+  );
+}
+
+if (targetId) {
+  prepareAnnotations();
+} 
+
+</script>
+
+<template>
+  <Loading v-if="loading" />
+  <div v-else>
+    <h4 class="mt-4">Selection</h4>
+    <WctTabViewPanel columns>
+      <div class="flex items-start justify-between gap-8 w-full">
+        <div class="flex flex-col items-start gap-2 w-full">
+          <WctFormField label="Selection date">
+            <p class="font-semibold">{{ formatDate(targetAnnotations.selection.date) }}</p>
```

Comment:
If the selection.date is null, 01/01/1970 is shown now. It's better to show an empty string.